service/ExchangeService.js

```diff
@@ -7,19 +7,31 @@ var btcPrice   = null;
 var krwUsd     = null;
 var log = new Logger(AppConfig.logLevel);
 
-
+var handleError = function(message, exchange, errortype){
+	var errorObject = new Error();
+		errorObject.message = message;
+		errorObject.exchange = exchange;
+		errorObject.errorType = errortype;
+	return errorObject;
+}
 var fetchExchangeData = function(exchange, url, callback){
 	request.get(url, function (err, response, body) {
 		if ( !err && response.statusCode == 200 ){
```

Comment:
Please try to avoid complex `if else` statements, especially `elseif`. It increases [cyclomatic complexity](https://en.wikipedia.org/wiki/Cyclomatic_complexity):
- https://softwareengineering.stackexchange.com/questions/255523/do-else-blocks-increase-code-complexity
- https://softwareengineering.stackexchange.com/questions/206816/clarification-of-avoid-if-else-advice
- https://softwareengineering.stackexchange.com/questions/122485/elegant-ways-to-handle-ifif-else-else

In this case, you can handle error first (more common way) and then handle result:
```js
if (err) {
  ...
  callback(error);
  return;
}

if (response.statusCode !== 200) { // actually all 200-299 codes are ok
   ...
   callback(error);
   return;
}
...
callback(null, data);
```